### DISCUSSION TITLE
怎么让less报错有提示

### DISCUSSION THREAD

**Discussion by isToThyMoon** [umijs/dumi#1447] [Q&A]
hi各位，寻求帮助。尝试用dumi开发组件库，如果less变量未定义，或者有其他less语法错误，dumi构建就会卡在50%，但是没有任何报错提示。是要额外针对less作语法检查还是在dumi中开启什么配置就好。

**Comment by PeachScript** [comment]
应该是 dumi 的 bug，有什么地方把错误吞掉了，近期会修复

**Reply by isToThyMoon** [reply]
感谢回复，请问有其他方式暂时规避下么，写less没有一点检查太痛苦了。

**Reply by PeachScript** [reply]
配置 `mfsu: false` 可以规避

**Reply by isToThyMoon** [reply]
可以规避 感谢🙏
